src/MVC/view/authorization/authorization.ts

```diff
@@ -6,6 +6,8 @@ const logUrl = 'http://127.0.0.1:5000/api/login';
 function authorization(parent: HTMLElement) {
```

Comment:
На 2ой строчке импорта, тс ругается на название файла в импорте('./**L**ogIn') 
>Already included file name 'e:/GoSport-RClone/src/mvc/view/authorization/LogIn.ts' differs from file name 'e:/GoSport-RClone/src/mvc/view/authorization/logIn.ts' only in casing. The file is in the program because: Imported via './LogIn' from file 'e:/GoSport-RSClone/src/mvc/view/authorization/NewUser.ts' Imported via './LogIn' from file 'e:/GoSport-RSClone/src/mvc/view/authorization/authorization.ts' Matched by include pattern '**/*'

В оригинале файл называется **l**ogIn.ts
Когда поменял импорты в файле _authorization.ts_ и _NewUser.ts_ на ('/**l**ogIn') ошибка пропала